pkg/adapter/metrics.go

```diff
@@ -19,11 +19,21 @@ import (
 	"time"
 )
 
+// Metric kinds supported by mixer.
 const (
-	// GAUGE is used to record instantaneous (non-cumulative) measurement
-	GAUGE Kind = iota
-	// COUNTER is used to record increasing cumulative values.
-	COUNTER
+	Gauge   Kind = iota // records instantaneous (non-cumulative) measurements
+	Counter             // records increasing cumulative values
+)
+
+// Label kinds supported by mixer.
+const (
```

Comment:
Any reason why this doesn't include all the types defined in AttributeDescriptor.ValueType?